src/karmabot/commands/score.py

```diff
@@ -1,49 +1,44 @@
-from karmabot.db import db_session
+import karmabot.slack
+from karmabot.db.database import database
 from karmabot.db.karma_user import KarmaUser
 
 TOP_NUMBER = 10
 
 
 def get_karma(**kwargs):
     """Get your current karma score"""
-    user_id = kwargs.get("user_id").strip("<>@")
+    user_id = kwargs.get("user_id")
+    slack_id = karmabot.slack.get_slack_id(user_id)
 
-    session = db_session.create_session()
-    kama_user = session.query(KarmaUser).get(user_id)
+    with database.session_manager() as session:
+        kama_user = session.query(KarmaUser).get(user_id)
 
-    try:
-        if not kama_user:
-            return "User not found"
+    if not kama_user:
+        return "User not found"
```

Comment:
I see a lot of string returns, would it be more idiomatic to raise an exception here?